vpx git: make failed

```
┌ vpx git ........................................... [Recently updated]
├ Running configure...
├ Running make...
Likely error:
{standard input}: Assembler messages:
{standard input}:19: Error: invalid register for .seh_savexmm
make[1]: *** [Makefile:157: vpx_dsp/x86/sad4d_avx512.c.o] Error 1
make[1]: *** Waiting for unfinished jobs....
make: *** [Makefile:17: .DEFAULT] Error 2
    [CC] vpx_dsp/x86/sad4d_avx512.c.o
{standard input}: Assembler messages:
{standard input}:19: Error: invalid register for .seh_savexmm
make[1]: *** [Makefile:157: vpx_dsp/x86/sad4d_avx512.c.o] Error 1
make: *** [Makefile:17: .DEFAULT] Error 2
make failed. Check D:/Tools/mabs/build/vpx-git/build-64bit/ab-suite.make.log
This is required for other packages, so this script will exit.
  Creating diagnostics file...

All relevant logs have been anonymously uploaded to https://0x0.st/sWfx.zip
Copy and paste [logs.zip](https://0x0.st/sWfx.zip) in the GitHub issue.
Make sure the suite is up-to-date before reporting an issue. It might've been fixed already.
Try running the build again at a later time.
```
[logs.zip](https://0x0.st/sWfx.zip)

--- a/build/media-suite_compile.sh
+++ b/build/media-suite_compile.sh
@@ -800,7 +800,7 @@ if [[ $vpx = y ]] && do_vcs "https://chromium.googlesource.com/webm/libvpx" vpx;
     [[ $ffmpeg = "sharedlibs" ]] || extracommands+=(--enable-{vp9-postproc,vp9-highbitdepth})
     extracommands+=($(get_external_opts))
     log "configure" ../configure --target="${arch}-win${bits%bit}-gcc" --prefix="$LOCALDESTDIR" \
-        --disable-{shared,unit-tests,docs,install-bins} \
+        --disable-{shared,unit-tests,docs,install-bins,avx512} \
         "${extracommands[@]}"
     for _ff in *.mk; do
         sed -i 's;HAVE_GNU_STRIP=yes;HAVE_GNU_STRIP=no;' "$_ff"

